master/getting-started/openshift/installation.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Installing Calico on OpenShift
+---
+
+Installation of Calico in OpenShift is integrated in openshift-ansible v3.6.
```

Comment:
I think it would be good to give a link to the OpenShift installation docs and make a statement about how the sections here should be used when using those docs.  I think this is especially important without an Overview page but is probably still good to have once there is an Overview page.